recipes-support/lvm2/lvm2_%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILES_libdevmapper += " \
```

Comment:
Why do we need this changes?

Reply:
By default libdevmapper doesn't add udev rules to the system. These udev rules are necessary for systems crypto tools synchronisation.